addons/website/static/src/services/website_service.js

```diff
@@ -176,23 +179,16 @@ export const websiteService = {
                     },
                 });
             },
-            async fetchWebsites() {
+            async fetchUserGroups() {
                 // Fetch user groups, before fetching the websites.
                 [isRestrictedEditor, isDesigner, hasMultiWebsites] = await Promise.all([
                     user.hasGroup('website.group_website_restricted_editor'),
                     user.hasGroup('website.group_website_designer'),
                     user.hasGroup('website.group_multi_website'),
                 ]);
-
-                const [currentWebsiteRepr, allWebsites] = await Promise.all([
-                    orm.call('website', 'get_current_website'),
-                    hasMultiWebsites ? orm.searchRead('website', [], ['domain', 'id', 'name']) : [],
-                ]);
-                websites = [...allWebsites];
-                setCurrentWebsiteId(unslugHtmlDataObject(currentWebsiteRepr).id);
-                if (!websites.length) {
-                    websites = [{ id: currentWebsiteId }];
-                }
+            },
+            async fetchWebsites() {
+                websites = [...(await orm.searchRead('website', [], ['domain', 'id', 'name']))];
```

Comment:
Mmh, you now do this RPC everytime even in the most common case of mono-website. But we can see what to do about that later, see other comment.